2020-06-04 01:00:53.1359 ERROR В методе: Void PrepareToSaveChanges(System.Data.Entity.Core.Objects.SaveOptions) возникло исключение: Операция завершилась с ошибкой. Не удалось изменить связь, поскольку один или несколько свойств внешнего ключа не допускают значения NULL. При изменении связи соответствующему свойству внешнего ключа присваивается значение NULL. Если внешний ключ не поддерживает значений NULL, должна быть определена новая связь, свойству внешнего ключа должно быть присвоено другое значение, отличное от NULL, либо необходимо удалить несвязанный объект.
2020-06-04 01:17:12.7191 ERROR В методе: Void PrepareToSaveChanges(System.Data.Entity.Core.Objects.SaveOptions) возникло исключение: Операция завершилась с ошибкой. Не удалось изменить связь, поскольку один или несколько свойств внешнего ключа не допускают значения NULL. При изменении связи соответствующему свойству внешнего ключа присваивается значение NULL. Если внешний ключ не поддерживает значений NULL, должна быть определена новая связь, свойству внешнего ключа должно быть присвоено другое значение, отличное от NULL, либо необходимо удалить несвязанный объект.
2020-06-04 01:22:40.9875 ERROR В методе: Void PrepareToSaveChanges(System.Data.Entity.Core.Objects.SaveOptions) возникло исключение: Операция завершилась с ошибкой. Не удалось изменить связь, поскольку один или несколько свойств внешнего ключа не допускают значения NULL. При изменении связи соответствующему свойству внешнего ключа присваивается значение NULL. Если внешний ключ не поддерживает значений NULL, должна быть определена новая связь, свойству внешнего ключа должно быть присвоено другое значение, отличное от NULL, либо необходимо удалить несвязанный объект.
